internal/business/persisted_operations/persisted_operations_test.go

```diff
@@ -127,10 +127,11 @@ func TestNewPersistedOperations(t *testing.T) {
 					assert.NoError(t, err)
 
 					assert.Equal(t, "query { foobar }", payload.Query)
-					assert.Equal(t, int64(101), r.ContentLength)
+					assert.Equal(t, int64(82), r.ContentLength)
```

Comment:
To verify, the contentLength here is smaller because the OperationName is no longer part of the response body, right?

Reply:
correct